models/lo_2010.cpp

```diff
@@ -26,12 +26,12 @@ double CLi = 1.0 * pow((wt/NORM_WT), 0.75) * pow((pma/NORM_PMA), 3.16) * (0.83*S
 double Vi = 0.572 * (1-0.344*spiro) * wt
 
 [OMEGA] @annotated
-  ECL : 0.04 : Eta on CL
-  EV  : 0.01 : Eta on V
+  ECL : 20.5 : Eta on CL
```

Comment:
what are all these values? they still need to be reported as variances, so you were correct in that 0.04 --> 20%CV. Remember the variance to %CV calculation can be approximated by sqrt(variance)*100, so sqrt(0.04)*100 = 20%